site/src/pages/WorkspacesPage/WorkspacesPageView.stories.tsx

```diff
@@ -265,3 +266,30 @@ export const InvalidPageNumber: Story = {
 		page: 1000,
 	},
 };
+
+export const ShowOrganizations: Story = {
+	args: {
+		workspaces: [
+			{
+				...MockWorkspace,
+				organization_name: "Limbus Co.",
+			},
+		],
+	},
+
+	parameters: {
+		showOrganizations: true,
+	},
+
+	play: async ({ canvasElement }) => {
+		const canvas = within(canvasElement);
+		const accessibleTableCell = await canvas.findByRole("cell", {
+			// Need whitespace classes because different parts of the element
+			// are going in different spans, and Storybook doesn't consolidate
+			// these
+			name: /org\s?(?:anization)?\s?:\s?Limbus Co\./i,
+		});
+
+		expect(accessibleTableCell).toBeDefined();
+	},
```

Comment:
Just wanted to make sure that I understand the limitations around decorators: they're basically assumed to always be static values, right? There's no easy way to make a decorator re-render with a new value?

Reply:
each "decorator" undergoes a small transformation to get turned into a component by storybook. there's no way to force a rerender from the outside, just like any other component. being a component tho, they can call hooks and have state, which is kind of non-obvious.